openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -60,32 +60,35 @@ This specification uses the terms "Access Token", "Authorization Endpoint", "Aut
 
 This specification also defines the following terms. In the case where a term has a definition that differs, the definition below is authoritative for this specification.
 
-Credential:
-:  A set of one or more claims about a subject made by a Credential Issuer. Note that this definition of a term "credential" in this specification is different from that in [@!OpenID.Core] and [@!RFC6749].
+Credential Dataset:
+:  A set of one or more claims about a subject made by a Credential Issuer.
 
-Verifiable Credential (VC):
-:  An Issuer-signed Credential whose integrity can be cryptographically verified. It can be of any format used in the Issuer-Holder-Verifier Model, including, but not limited to those defined in [@VC_DATA] and [@ISO.18013-5].
```

Comment:
As far as I can see, the current text uses the terms Credential and Verifiable Credential interchangeably. I also question whether we have a use for the non-verifiable Credential term. So I suggest to adapt the terminology accordingly.

Reply:
I had the same observation. I agree to not differentiate.

Reply:
So what is the suggestion? To always use "Verifiable Credential"?

Reply:
Always use Credential, just as it is in the text right now.